backend/union/policy/eplno.go

```diff
@@ -18,15 +18,28 @@ type EpLno struct {
 	EpAll
 }
 
-func (p *EpLno) lno(upstreams []*upstream.Fs) (*upstream.Fs, error) {
+func (p *EpLno) getUpstreamObjectCount(ctx context.Context, upstream *upstream.Fs) int64 {
+	numObj, err := upstream.GetNumObjects()
+	if err != nil {
+		uName := upstream.Name()
+		fs.LogPrintf(fs.LogLevelNotice, nil,
+			"Number of Objects is not supported for upstream %s, falling back to listing (this may be slower)...", uName)
+		listing, err := upstream.List(ctx, "")
```

Comment:
This should use this function

https://github.com/rclone/rclone/blob/e0c99d620340f974255820882ab2802c6694a9ab/fs/operations/operations.go#L1009-L1012

Otherwise you've just listed the root directory which I don't think is what you want to do